### DISCUSSION TITLE
ORA - 29024 Certificate Validation Failure

### DISCUSSION THREAD

**Discussion by harshshah1618** [oracle/python-oracledb#299] [Q&A] [answered]
I am trying to connect via thick client and have all necessary files and folders in place.
sqlnet.ora , tnsnames.ora and cwallet.sso are in the admin folder
But when i try to call oracledb.init_oracle_client() i get the above error
Can you please help?

**Comment by anthony-tuininga** [comment, marked as answer]
That error suggests that you have the *wrong* cwallet.sso file in place. You can try connecting to the database using SQL*Plus and the same configuration -- that should fail with the same error.

**Reply by harshshah1618** [reply]
Thanks
It was the wallet issue.
Fixed it
